enigma-swing/src/main/java/cuchaz/enigma/gui/config/KeyBindsConfig.java

```diff
@@ -0,0 +1,50 @@
+package cuchaz.enigma.gui.config;
+
+import cuchaz.enigma.config.ConfigContainer;
+import cuchaz.enigma.config.ConfigSection;
+import cuchaz.enigma.gui.config.keybind.KeyBind;
+
+import java.util.Optional;
+
+public final class KeyBindsConfig {
+    private KeyBindsConfig() {
+    }
+
+    private static final ConfigContainer cfg = ConfigContainer.getOrCreate("enigma/enigmakeybinds");
+
+    private static ConfigSection runningCfg;
+
+    static {
+        KeyBindsConfig.snapshotConfig();
+    }
+
+    // Save the current configuration state so consistent keybinds can be provided
+    // for keybinds that are not configurable during runtime.
+    public static void snapshotConfig() {
```

Comment:
Hmmm, is it not possible to change all these in the UI immediately when set through the options? Need to take a closer look later

Reply:
I managed to make it work without issues